installer/dev/install.cpp

```diff
@@ -186,7 +186,7 @@ namespace WindowsAppRuntimeInstaller
         // Populate framework from the manifest properties.
         wil::com_ptr<IAppxManifestProperties> manifestProperties;
         THROW_IF_FAILED(manifest->GetProperties(wil::out_param(manifestProperties)));
-        BOOL isFramework = FALSE;
+        BOOL isFramework{ FALSE };
```

Comment:
NIT: `BOOL isFramework{};`

No need for FALSE. Think of `{}` as "call the void ctor" and even primitive types have a void ctor (essentially) -- false/0/0.0/null/etc

Same comment applies throughout